ubuntu/18.04/extra-packages

```
@@ -6,3 +6,4 @@ unzip
 zip
 zsh
 curl
+libcap2-bin
```

You could even go so far as to add it to every image.  I suppose, if the package is already present, then it will get silently skipped.

I don't know how `capsh(1)` went missing in the first place, because I assume that the image did work at some point.  It could be that the dependency chains between the packages changed at some point, and `capsh(1)` fell off the image by mistake.  Being defensive about the packages that we definitely want will avoid these problems.

---

Good point. I'll add it to all images